spec/system/group_spec.rb

```diff
@@ -6,7 +6,7 @@
   before(:all) do
     @user = FactoryBot.create(:user)
     @user2 = FactoryBot.create(:user)
-    @group = FactoryBot.create(:group, mentor: @user)
+    @group = FactoryBot.create(:group, mentor: @user, name: "CircuitVerse")
```

Comment:
Since you need name to be present to get a valid group factory, add name in 
factory definition instead of duplicating the logic everywhere